tracekit.js

```diff
@@ -1225,8 +1225,13 @@ if (!TraceKit.linesOfContext || TraceKit.linesOfContext < 1) {
 }
 
 
-
-// Export to global object
-window.TraceKit = TraceKit;
+// UMD export
+if (typeof module != 'undefined' && module.exports && this.module !== module) {
+    module.exports = TraceKit;
+} else if (typeof define === 'function' && define.amd) {
+    define(TraceKit);
+} else {
+    window.TraceKit = TraceKit;
+}
 
 }(typeof window !== 'undefined' ? window : global));
```

Comment:
I just think it would be nice for CommonJS user to use TraceKit like 

```javascript
import TraceKit from 'tracekit';
TraceKit.blah();
```

instead of 

```javascript
import 'traceket';
window.TraceKet.blah();
```

but it's all up to you.